packages/lesswrong/server/ckEditorToken.ts

```diff
@@ -25,7 +26,7 @@ export async function ckEditorTokenHandler (req, res, next) {
 
   const user = await getUserFromReq(req)
   const post = documentId && await Posts.findOne(documentId)
-  const canEdit = post && postCanEdit(user, post)  
+  const canEdit = post && (userOwns(user, post) || userCanDo(user, 'posts.edit.all') || await userIsPostGroupOrganizer(user, post))
```

Comment:
I'm not totally sure how to test this change, but I tested editing an event (created by a different user) as a group organizer and that worked, so I think that covers this?

Reply:
This stuff is for the collaborative editor integration, which I'm working on in another branch.